client/src/components/SitterProfile/useStyles.ts

```diff
@@ -0,0 +1,64 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  avatar: {
+    height: '110px',
+    width: '110px',
+    marginTop: theme.spacing(-6),
+    backgroundColor: 'white',
+    padding: theme.spacing(1),
+    boxShadow: '0px 0px 10px -3px rgba(0,0,0,0.45)',
+  },
+  mainText: {
+    color: 'black',
+    fontWeight: 600,
+  },
+  description: {
+    fontSize: 13,
```

Comment:
You can leverage MUI variants (after setting a font size to each variant in MUI's theme) instead of manually assigning font sizes to each element :)

Reply:
Implemented variants for the listing + profile page. 
Much cleaner and more consistent. Thanks